php/Terminus/Outputters/PrettyFormatter.php

```diff
@@ -21,7 +21,7 @@ class PrettyFormatter implements OutputFormatterInterface {
    */
   public function formatValue($value, $human_label = '') {
     $value = PrettyFormatter::flattenValue($value);
-    return $human_label ? "$human_label: $value\n" : $value;
+    return $human_label ? "$human_label: $value" . PHP_EOL : $value;
```

Comment:
Both this style of if-else statement and returning values before assigning them to a variable are forbidden by Terminus' PHP standards.